common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -665,6 +665,16 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String NEGOTIATED_BP_VERSION_KEY = "common.broker.protocol.version.name";
 
+        /**
+         * Code of the error that occurs as a result of MSAL-Broker protocol handshake (hello()).
+         */
+        public static final String HELLO_ERROR_CODE = "error";
```

Comment:
introduced new variables, same values, because using OAUTH variables in hello is really confusing.